include/SFML/Graphics/Shape.hpp

```diff
@@ -304,15 +304,15 @@ class SFML_GRAPHICS_API Shape : public Drawable, public Transformable
     ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    const Texture* m_texture;          //!< Texture of the shape
-    IntRect        m_textureRect;      //!< Rectangle defining the area of the source texture to display
-    Color          m_fillColor;        //!< Fill color
-    Color          m_outlineColor;     //!< Outline color
-    float          m_outlineThickness; //!< Thickness of the shape's outline
-    VertexArray    m_vertices;         //!< Vertex array containing the fill geometry
-    VertexArray    m_outlineVertices;  //!< Vertex array containing the outline geometry
-    FloatRect      m_insideBounds;     //!< Bounding rectangle of the inside (fill)
-    FloatRect      m_bounds;           //!< Bounding rectangle of the whole shape (outline + fill)
+    const Texture* m_texture{nullptr};           //!< Texture of the shape
+    IntRect        m_textureRect;                //!< Rectangle defining the area of the source texture to display
+    Color          m_fillColor{Color::White};    //!< Fill color
+    Color          m_outlineColor{Color::White}; //!< Outline color
+    float          m_outlineThickness{0};        //!< Thickness of the shape's outline
+    VertexArray    m_vertices{PrimitiveType::TriangleFan};          //!< Vertex array containing the fill geometry
+    VertexArray    m_outlineVertices{PrimitiveType::TriangleStrip}; //!< Vertex array containing the outline geometry
+    FloatRect      m_insideBounds;                                  //!< Bounding rectangle of the inside (fill)
+    FloatRect      m_bounds; //!< Bounding rectangle of the whole shape (outline + fill)
```

Comment:
I don't know if it's an issue, but on m_bounds, is there consistent indention for the comments?

Reply:
This is a quirk of clang-format trying to maintain vertical alignment with exceeding out specified line width. Not much we can do about it without disabling certain vertical alignment rules across the codebase.